### DISCUSSION TITLE
to continue #393

### DISCUSSION THREAD

**Discussion by LouisBeke2** [ArugaZ/whatsapp-bot#395] [Q&A] [answered]
if i do `yarn generateLang` i get this error 

Error: Invalid or incompatible cached data (cachedDataRejected)
    at L.<computed>.<computed> [as .arugaz] (C:\Users\louis\Desktop\whatsapp-bot\generateLang:2:10493)
    at Module.load (node:internal/modules/cjs/loader:1037:32)
    at Module._load (node:internal/modules/cjs/loader:878:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
    at node:internal/main/run_main_module:23:47

Node.js v18.12.1
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

can you help

**Comment by arugaz** [comment, marked as answer]
> if i do `yarn generateLang` i get this error
> 
> Error: Invalid or incompatible cached data (cachedDataRejected) at L.. [as .arugaz] (C:\Users\louis\Desktop\whatsapp-bot\generateLang:2:10493) at Module.load (node:internal/modules/cjs/loader:1037:32) at Module._load (node:internal/modules/cjs/loader:878:12) at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12) at node:internal/main/run_main_module:23:47
> 
> Node.js v18.12.1 error Command failed with exit code 1. info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
> 
> can you help

Thank for reporting, i got same error on my windows machine, I'll update it later.

@LSystemus-2 Try with
```
yarn generateLangV2
```

**Reply by LouisBeke2** [reply]
can i have something to report the bug in prived?

**Reply by arugaz** [reply]
> can i have something to report the bug in prived?

Sure! chat me on insta / whatsapp

**Reply by LouisBeke2** [reply]
okey
